autoplex/data/common/jobs.py

```diff
@@ -611,14 +641,21 @@ def VASP_collect_data(
     return {"vasp_ref_dir": vasp_ref_dir, "isol_es": isol_es}
 
 
-def check_convergence_vasp(file):
+def check_convergence_vasp(path: str) -> bool:
```

Comment:
In the long run, we should rely on custodian handlers again. This is something that can be checked within custodian.

Reply:
We've implemented the custodian handlers now, so I'll go ahead and remove the function.